src/components/PatentUpload/PatentUpload.css

```diff
@@ -0,0 +1,29 @@
+label {
+    margin-top: 25px;
+    display:block;
+    font-size: medium;
+    font-weight: 500;
+}
+
+input {
```

Comment:
once the app is build all the css files will be merged together, we should not rely on styling on a type of element as that will impact the entire page. The best two options imo are:

1) use css class following BEM so something like: patent-upload__pdf-input this way we make sure there are no conflicting styles
2) If the styles scope is narrow using inline styles is also ok so in JSX (react) you can do:
`<input style={{ marginBottom: 50 }}>` note that instead of using `-` like css it uses camelCase, react does the convert for you.